language-plutus-core/src/Language/PlutusCore/Constant/Apply.hs

```diff
@@ -79,6 +82,10 @@ instance ( PrettyBy config (Constant ())
         [ "A non-constant argument of a sized type: "
         , prettyBy config arg
         ]
+    prettyBy config (UnreadableBuiltinConstAppError arg)    = fold
+        [ "Could not construct denotation for a built-in:", "\n"
```

Comment:
If you're using a pretty-printer, you probably want to use something like `hardline` instead of `"\n"`.

Reply:
Yeah. But there are other pieces with `"\n"`, I don't want to refactor stuff in this PR. I'll do that later, thanks.